.github/workflows/pipe-it-2cluster.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         # StrongConsistencyClusterMode is ignored now because RatisConsensus has not been supported yet.
         cluster1: [LightWeightStandaloneMode, ScalableSingleNodeMode, HighPerformanceMode]
         cluster2: [LightWeightStandaloneMode, ScalableSingleNodeMode, HighPerformanceMode]
-    runs-on: [self-hosted, iotdb]
```

Comment:
matrix:
        os: [ ubuntu-latest, windows-latest ]
    runs-on: ${{ matrix.os }}
    
    runs-on should remove self-hosted and iotdb

Reply:
Here I just want to use github machines when self-hosted machines are unavailable. Yet in github documents(the link I pasted on the description), I failed to find a precise example. Do you have any ideas?